climate_scanner/trends_innovation_classifier/test/test_doc_to_sentence.py

```diff
@@ -0,0 +1,18 @@
+import unittest
```

Comment:
I tend to use pytest rather than unittest, it has a really nice interface. If you're keen to use this no worries though.

Reply:
Sure we can keep it consistent. I used pytest too in the past, will shift to pytest moving forward.